plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1283,8 +1283,16 @@ def update_auth_policy_server(self, ipAddress):
                 ise_state_set = {"FAILED", "INPROGRESS"}
                 state = ise_server_details.get("state")
                 if state in ise_state_set:
-                    self.msg = "The Cisco ISE server '{0}' integration is not successful. The state is '{1}'. ".format(ipAddress, state) + \
-                               "Expected states for successful integration are not in {0}.".format(ise_state_set)
+                    if state == "INPROGRESS":
+                        self.msg = "The Cisco ISE server '{0}' integration is not completed. The state is 'INPROGRESS'. ".format(ipAddress) + \
+                                   "Integration took more than '{0}' seconds.".format(ise_integration_wait_time)
+                    elif state == "FAILED":
+                        self.msg = "The Cisco ISE server '{0}' integration is not successful. The state is 'FAILED'." \
```

Comment:
self.msg = ("The Cisco ISE server '{ip}' integration has failed and is in 'FAILED' state."
                    .format(ip=ipAddress))